### ISSUE TITLE
Pre-defined task improvements

### ISSUE THREAD

**Issue by JulienVig** [epfml/disco#662]
- [x] Training when a saved model already exists reuses it by default, while it should create a new model overwrite it.
- [x] The training graphs should show the training and validation loss too, sometimes the accuracies stay constant while the loss still changes.
- [x] The test accuracy graph doesn't update and stays at 0 -> #678 removed the test line plot (because it was only plotting a single scalar value)
- [x] The testing an image model (e.g. lus_covid) shows what are the predictions for each image and where the model made classification mistakes (groundtruth vs prediction). However errors are sometimes not displayed (only prediction).
- [ ]  Validation set sometimes contains only one class
    Image tasks support choosing a train-validation split in which a percentage of the training set is set apart as a validation set (`discojs-core/src/dataset/data_loader/image_loader.ts`). In some cases however, e.g., when using imbalanced or small datasets, this can result in some classes not being included in the validation set at all.
Unfortunately, it seems that tensorflow.js doesn't offer any function for stratified splits.
- [x] Cached model is used by default while it is said in the task that it would be overwritten. Moreover the toggle button to disable this isn't displayed.
